Buildings/Experimental/DHC/CentralPlants/Cooling/Plant.mo

```diff
@@ -0,0 +1,383 @@
+within Buildings.Experimental.DHC.CentralPlants.Cooling;
+model Plant
+  "District cooling plant model"
+  extends Buildings.Experimental.DHC.CentralPlants.BaseClasses.PartialPlant(
+    have_eleCoo=true,
+    have_pum=true,
+    have_fan=true,
+    have_weaBus=true,
+    typ=Buildings.Experimental.DHC.Types.DistrictSystemType.Cooling);
+  // chiller parameters
+  parameter Integer numChi(
+    min=1,
+    max=2)=2
+    "Number of chillers, maximum is 2"
```

Comment:
I think any other numbers than 2 would break the model, given that the `ChillerStage` control block is harcoded for 2 chillers.

Reply:
Agree, we have hardcoded the number of chillers to 2.